src/worker/course.ts

```diff
@@ -29,6 +29,20 @@ function toRadians(value: number) {
   return (value * Math.PI) / 180
 }
 
+/** Normalises angle to a value within the range of a compass
+ * @param angle: angle (in radians or degrees)
+ * @param units: unit of the angle supplied => 'rad' (radians) or 'deg' (degrees)
+ * @returns value between 0 - 2*PI
+ */
+function compassAngle(angle: number, units: 'rad' | 'deg' = 'rad'): number {
```

Comment:
Why include degrees support? Afai can tell this function is local and never used with anything but radians.

Reply:
Removed units so it is just radians.